tests/test_id_generator.py

```diff
@@ -0,0 +1,11 @@
+from bubuku.id_generator import BrokerIDByIp
+from test_config import build_test_properties
+
+
+def test_max_id_replaced_on_ip():
+    kafka_properties = build_test_properties()
+    assert kafka_properties.get_property('reserved.broker.max.id') is None
+    BrokerIDByIp(None, '192.168.1.1', kafka_properties)
```

Comment:
For me test doesn't work because of this None
```
>       BrokerIDByIp(None, '192.168.1.1', kafka_properties)
E       TypeError: __init__() takes 3 positional arguments but 4 were given

tests/test_id_generator.py:8: TypeError
```